product_docs/docs/pem/8.0/pem_online_help/04_toc_pem_features/12_pem_manage_probes/01_pem_custom_probes.mdx

```diff
@@ -19,19 +19,19 @@ Use the fields on the `General` tab to modify the definition of an existing prob
 -   Use the `Probe name` field to provide a name for a new probe.
 -   Use the `Collection method` field to specify the probe type. Use the drop-down listbox to select from:
 
-> -   `SQL` (the probe will gather information via a SQL statement)
->
-> -   `WMI` (the probe will gather information via a Windows Management Instrumentation extension)
->
-> -   `Batch/Shell Script` (the probe will use a command-script or shell-script to gather information).
->
->     Before creating a batch probe on a Linux system, you must modify the `agent.cfg` file, setting the `allow_batch_probes` parameter equal to `true` and restart the PEM agent. The `agent.cfg` file is located in `/opt/PEM/agent/etc`.
->
->     On 64-bit Windows systems, agent settings are stored in the registry. Before creating a batch probe, modify the registry entry for the `AllowBatchProbes` registry entry and restart the PEM agent. PEM registry entries are located in `HKEY_LOCAL_MACHINE\\Software\\Wow6432Node\\EnterpriseDB\\PEM\\agent`.
->
->     Please note that batch probes are platform-specific. If you specify a collection method of `Batch`, you must specify a platform type in the `Platform` field.
->
->     To invoke a script on a Linux system, you must modify the entry for `batch_script_user` parameter of agent.cfg file and specify the user that should be used to run the script. You can either specify a non-root user or root for this parameter. If you do not specify a user, or the specified user does not exist, then the script will not be executed. Restart the agent after modifying the file. If pemagent is being run by a non-root user then the value of `batch_script_user` will be ignored and the script will be executed by the same non-root user that is being used for running the pemagent.
+-   `SQL` (the probe will gather information via a SQL statement)
+
+-   `WMI` (the probe will gather information via a Windows Management Instrumentation extension)
+
+-   `Batch/Shell Script` (the probe will use a command-script or shell-script to gather information).
+
+    Before creating a batch probe on a Linux system, you must modify the `agent.cfg` file, setting the `allow_batch_probes` parameter equal to `true` and restart the PEM agent. The `agent.cfg` file is located in `/opt/PEM/agent/etc`.
+
+    On 64-bit Windows systems, agent settings are stored in the registry. Before creating a batch probe, modify the registry entry for the `AllowBatchProbes` registry entry and restart the PEM agent. PEM registry entries are located in `HKEY_LOCAL_MACHINE\\Software\\Wow6432Node\\EnterpriseDB\\PEM\\agent`.
```

Comment:
Please check if one backslash is extra in the path. I think it should be "HKEY_LOCAL_MACHINE\Software\WOW6432Node..."

Reply:
Fixed this.